jobs/tcp_router/spec

```diff
@@ -59,13 +59,31 @@ properties:
       be set. For mTLS also set tcp_router.backend_tls.client_cert and
       tcp_router.backend_tls.client_key.
     default: false
+  tcp_router.frontend_tls_pem.enabled:
```

Comment:
This isn't mentioned in the tcp router ERB. Should it be added there, or removed here?